app/views/orders/index.html.erb

```diff
@@ -1,8 +1,10 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Listing Orders</h1>
+<h1>Текущие заказы</h1>
 
-<table>
+  <table id="data-table" border ="1" rules ='rows'></table>
+<script type="text/javascript" src="myScript/orders.js"></script>
+<!-- <table id="table-order">
```

Comment:
Please, don't comment code, just delete it.
We use git, so we would restore it if we need it.